perllib/FixMyStreet/App/Controller/Offline.pm

```diff
@@ -35,10 +35,17 @@ sub fallback : Local {
     my ($self, $c) = @_;
 }
 
-sub manifest: Path("/.well-known/manifest.webmanifest") {
-    my ($self, $c) = @_;
+sub manifest: Path("/.well-known/manifest.webmanifest") : Args(1) {
```

Comment:
I think we want to maintain a recognised file extension (I agree it shouldn't matter, but MDN at https://developer.mozilla.org/en-US/docs/Web/Manifest does say "Browsers generally support manifests with other appropriate extensions" which might imply they don't support things without, though I assume they are fine) - so could this be /.well-known/manifest-fms.webmanifest' and manifest-waste perhaps? Might need two tiny controllers that then call a shared function for the bulk